bitwuzla/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+add_library(smt-switch-bitwuzla "${SMT_SWITCH_LIB_TYPE}"
+  "${CMAKE_CURRENT_SOURCE_DIR}/src/bitwuzla_factory.cpp"
+  "${CMAKE_CURRENT_SOURCE_DIR}/src/bitwuzla_solver.cpp"
+  "${CMAKE_CURRENT_SOURCE_DIR}/src/bitwuzla_sort.cpp"
+  "${CMAKE_CURRENT_SOURCE_DIR}/src/bitwuzla_term.cpp"
+  )
+
+target_include_directories (smt-switch-bitwuzla PUBLIC "${PROJECT_SOURCE_DIR}/include")
+target_include_directories (smt-switch-bitwuzla PUBLIC "${PROJECT_SOURCE_DIR}/contrib/memstream-0.1/")
+target_include_directories (smt-switch-bitwuzla PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
+target_include_directories (smt-switch-bitwuzla PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/bitwuzla/include")
+target_include_directories (smt-switch-bitwuzla PUBLIC "${BITWUZLA_HOME}/src/api/c")
+
+target_link_libraries(smt-switch-bitwuzla "${BITWUZLA_HOME}/build/lib/libcbitwuzla.a")
+target_link_libraries(smt-switch-bitwuzla "${BITWUZLA_HOME}/deps/cadical/build/libcadical.a")
```

Comment:
Just FYI, for skeleton preprocessing, Bitwuzla still needs Lingeling (as does Boolector). We plan to switch this to CaDiCaL, but right now it isn't yet. If Bitwuzla is not built with Lingeling, skeleton preprocessing will not be performed.

Reply:
Thanks! What is skeleton preprocessing? It's probably okay to go without it until it's moved to CaDiCaL right?

Reply:
It's pretty much what the non-clausal simplification preprocessing pass does in CVC4. It's a preprocessing technique to derive simplifications on the Boolean layer.

Reply:
Performance should be better, in general, with skeleton preprocessing.